packages/react/src/select/root/SelectRoot.tsx

```diff
@@ -406,6 +413,7 @@ export function SelectRoot<Value, Multiple extends boolean | undefined = false>(
       disabled,
       readOnly,
       multiple,
+      serializedValue,
```

Comment:
This should be placed in the `store` sync effect where `store.update(...)` is called (like `value` is).
Also it seems `serializedValue: createSelector((state: State) => {` should be removed.